### PR TITLE
Add initial configuration

### PR DESCRIPTION
**Pull request by bedrich-schindler**

### CODE REVIEW

File: .idea/apache-php.iml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (fousjiri) [inline .idea/apache-php.iml]: This file should be probably in .gitignore.

File: php/php.ini [hunk 2]
```diff
@@ -0,0 +1,8 @@
+date.timezone = "Europe/Prague";
+
+memory_limit = 256M
```
Comment (fousjiri) [inline php/php.ini]: I would prefer to increase these limits at least to 512MB (maybe rather 1G) for running scripts and 128MB for upload.

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+###############
+# Install NPM #
+###############
+
+            && curl -sL https://deb.nodesource.com/setup_8.x | bash - \
+            && apt install -y nodejs \
+
+################################
+# Install Grunt, Gulp, Webpack #
```
Comment (fousjiri) [inline Dockerfile]: What about Robo task runner? Shouldn't it be here?

Comment (bedrich-schindler) [reply Dockerfile]: Nope as we have decided internally, Robo will no be installed globally anymore (Drush as well).

Comment (mbohal) [reply Dockerfile]: @fousjiri  no, we decided to have it as a project dependency defined in `composer.json`

**Pull request merged**